### DISCUSSION TITLE
Is manual file addition possible for Gokapi?

### DISCUSSION THREAD

**Discussion by hahyes** [Forceu/Gokapi#22] [Q&A] [answered]
Hey there,

I'm really interested into using Gokapi to serve downloading files I wanna to share. Sadly, upload through web is not what I'm looking for. I want to link files on server for Gokapi to share. Is this somehow possible? Just skip upload process and link to some path on server for Gokapi?

**Comment by Forceu** [comment, marked as answer]
It is possible to use the API for that, so you could post a local file with curl to the Gokapi service. Serving of local files is not supported and most likely will not be in the future.

For more information about the API, have a look at https://gokapi.readthedocs.io/en/stable/advanced.html#api
